tests/benchmarks/icfp15/pos/CopyRec.hs

```diff
@@ -49,33 +49,33 @@ data FHandle = FH Int deriving Eq
 
 {- ** API ** -}
 {-@ contents ::
-  d:FHandle -> RIO<{v:World | Lst v d},\w1 x -> {v:World | NoChange w1 v}> [Path]
+     d:FHandle -> RIO<{v:World | Lst v d},\w1 x -> {v:World | NoChange w1 v}> [Path]
```

Comment:
These are indented because otherwise liquid would complain.  Not sure why they worked in the previous test framework.